tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noImplicitAny": false,
```

Comment:
I don't think you need `noImplicitAny` when `strict` is true.

Reply:
It's meant to disable it ;)